Bundler/Viewer/Unity/Assets/Scripts/Network/Scripts/NetworkAnchorManager.cs

```diff
@@ -0,0 +1,487 @@
+        }
+    }
+
+    public override string ToString()
+    {
+        return String.Format("(anchor source ip: {0}) (anchor id: {1})", SourceIp, AnchorId);
+    }
+}
+
+public class LastReceivedAnchorArgs
+{
+    public LastReceivedAnchorArgs(WorldAnchorTransferBatch transferBatch)
+    {
+        TransferBatch = transferBatch;
+    }
+
+    /// <summary>
+    /// The transfer batch containing the anchor
+    /// </summary>
+    public WorldAnchorTransferBatch TransferBatch { get; private set; }
+
+    /// <summary>
+    /// The first anchor id in the transfer batch
+    /// </summary>
+    public string AnchorId
+    {
+        get
+        {
+            if (TransferBatch == null)
+            {
+                return String.Empty;
+            }
+
+            var ids = TransferBatch.GetAllIds();
+            if (ids == null || ids.Length == 0)
+            {
+                return String.Empty;
+            }
+
+            return ids[0];
+        }
+    }
+}
+
+public class NetworkAnchorManager : NetworkBehaviour
+{
+    /// <summary>
+    /// Get the local instance of the anchor manager
+    /// </summary>
+    private static NetworkAnchorManager instance;
+    public static NetworkAnchorManager Instance
+    {
+        get
+        {
+            return instance;
+        }
+    }
+
+    /// <summary>
+    /// This event is raised when a new anchor arrives from a different player.
+    /// </summary>
+    /// <param name="args">Contains the data that arrived.</param>
+    public delegate void OnLastReceivedAnchor(NetworkAnchorManager sender, LastReceivedAnchorArgs args);
+    public event OnLastReceivedAnchor LastReceivedAnchorChanged;
+
+    /// <summary>
+    /// Get if the local anchor player is in the process of receiving a shared anchor.
+    /// </summary>
+    public bool LoadingAnchor
+    {
+        get;
+        private set;
+    }
+
+    /// <summary>
+    /// Get the last received remote anchor
+    /// </summary>
+    public LastReceivedAnchorArgs LastReceivedAnchor
+    {
+        get;
+        private set;
+    }
+
+    /// <summary>
+    /// Get the local IP address.
+    /// </summary>
+    public String LocalAddress
+    {
+        get;
+        private set;
+    }
+
+    /// <summary>
+    /// The current source of the shared anchor 
+    /// </summary>
+    [SyncVar(hook = "ImportAnchorData")]
+    public SharedAnchorData AnchorSource;
+
+    /// <summary>
+    /// Get if the anchor is currently owned by a player.
+    /// </summary>
+    public bool IsSharedAnchorOwned
+    {
+        get
+        {
+            return AnchorSource.IsValid;
+        }
+    }
+
+    /// <summary>
+    /// This will send or receive the binary anchor data on the local instance
+    /// </summary>
+    private GenericNetworkTransmitter anchorTransmitter;
+
+    /// <summary>
+    /// Retry failed exports every second.
+    /// </summary>
+    private const float retryExportDelaySeconds = 1.0f;
```

Comment:
millisecond delay has been used elsewhere in the PR, why seconds here?

Reply:
"won't fix" .   The other were used to .NET APIs which expected MS.  This const is used with Unity APIs which expected Seconds.   Avoiding conversions.